models/user.py

```diff
@@ -7,15 +7,29 @@ class User(BaseModel):
     KEY_NAME_FIELDS = ['user_id']
 
     user_id = ndb.StringProperty(required=True)
-    name = ndb.StringProperty(indexed=False)
+    first_name = ndb.StringProperty(indexed=False)
+    last_name = ndb.StringProperty(indexed=False)
 
 
-def create_user(user_id, name=None):
+def create_user(user_id, first_name=None, last_name=None):
     if not user_id:
         raise ValueError('user_id is required')
 
     key = User.build_key(user_id=user_id)
-    new_user = User(key=key, user_id=user_id, name=name)
+    new_user = User(key=key, user_id=user_id, first_name=first_name, last_name=last_name)
     new_user.put()
 
-    return new_user
\ No newline at end of file
+    return new_user
+
+
+def update_user(user_id, first_name=None, last_name=None):
+    if not user_id:
+        raise ValueError('user_id is required')
```

Comment:
plz leave a line after an inner code block (like after loops or conditionals), just helps break it up and make it easy to read
```
if not user_id:
    raise ValueError('user_id is required')

key = User.build_key(user_id=user_id)
```